src/de/jost_net/JVerein/gui/control/ZusatzbetragVorlageControl.java

```diff
@@ -292,6 +308,15 @@ public void handleAction(Object context) throws ApplicationException
       zusatzbetraegeList.addColumn("Buchungstext", "buchungstext");
       zusatzbetraegeList.addColumn("Betrag", "betrag",
           new CurrencyFormatter("", Einstellungen.DECIMALFORMAT));
+      zusatzbetraegeList.addColumn("Zahlungsweg", "zahlungsweg", new Formatter() {
+        @Override
+        public String format(Object o)
+        {
+          if(o == null)
+            return "";
+          return new Zahlungsweg((Integer)o).getText();
+        }
+      });
```

Comment:
Könnte man hier noch die Buchungsklasse und die Buchungsart anzeigen?
Bei der Buchungsklasse abhängig vom Schalter in den Einstellungen, siehe Tabelle bei den Buchungen.